blobfuse/src/BlobfuseGcCache.cpp

```diff
@@ -164,6 +164,19 @@ void gc_cache::run_gc_cache()
             {
                 //clean up the file from cache
                 int fd = open(mntPath, O_WRONLY);
+                if (errno == 13) {
```

Comment:
errno = 13 : then open was not successful so there is no need to close

Reply:
done

Reply:
done